src/content/en/updates/2016/12/chrome-56-deprecations.md

```diff
@@ -0,0 +1,140 @@
+
+However, since touch events cannot support hover, hovering mousemoves followed a separate path. The whole design had quite a few side-effects including mouse interactions firing `TouchEvents`, all moue buttons appearing as *left* mouse buttons, and `MouseEvents` being suppressed by `TouchEvents`.
+
+Starting with Chrome 56, a mouse on Android M Or later will:
+
+* No longer fire `TouchEvents`.
+* Fire a cosistent sequence of `MouseEvents` with appropriate buttons and other properties. 
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/cNaFvMaYtNA/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5642080642662400) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=468806)
+
+## Remove user gestures from touch scroll
+
+We've seen multiple [examples](http://crbug.com/572319) of poorly written or malicious ads that trigger navigation for touch scrolls either on `touchstart` or all `touchend` events.  If a 'wheel' event can't open a pop-up, then touch scrolling shouldn't either. This may break some scenarios, for example, media not playing on touch, or pop-ups not opening on touch.  Safari already silently fails to open pop-ups in all of these scenarios.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/TO_x7FRkdmw/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/6131337345892352) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=611981)
+
+## Disallow all fetches for scripts with invalid type/language attributes
+
+Currently, Chrome's preload scanner fetches items in `<scripts>` elements regardless of the value of the `type` or `language` attribute, though the script will not be executed when parsed. By deprecating the fetch, the preload scanner and the parser will have the same semantics, and we will not be initiating fetches for scripts we will not use. This is intended to save data for users who navigate to sites with a lot of custom script tags that are post-processed (like `type=”text/template”`, for example).
+
+The use case of using invalid scripts to ping servers is adequately covered by the [sendBeacon API](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon).
+
+This change alligns Chrome with Safari, though Firebox still requests scripts regardless of type or language.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/eu57SCNltls/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5760718284521472) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=626321&desc=2)
+
+## Remove MediaStreamTrack.getSources()
+
+This method is no longer part of the spec and is not supported by any other major browser. It has been replaced by `MediaDevices.enumerateDevices()`, which Blink has supported without flags since version 47 and which is also supported by other browsers.
```

Comment:
Maybe link to https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/enumerateDevices